### PR TITLE
new UI vr 1

### PR DESCRIPTION
**Pull request by IshaanDesai45**
cards on the landing page changed,added a new layout.

### CODE REVIEW

**Comment by thebishaldeb** [comment]
Resolve the conflicts and upload a screenshots of the new layout in the PR description.

**Comment by IshaanDesai45** [comment]
instance
instance2

here i have attached the screenshots for the new layout 
I have changed the cards in the on the landing page as well ...And one image error is shown in the card is because of faulty url in my configured database

File: public/stylesheets/main.css [hunk 1]
```diff
@@ -167,7 +167,11 @@ margin-left: 15px;
     width: 60px;
   }
   body{
-    background-image: url("profilebg.jpg");
+    /*background-color: red;*/
+    background-image:url(ishaan.jpg);
```
Comment (thebishaldeb) [inline public/stylesheets/main.css]: Do not keep your name as the background picture. It should be something different related to the webpage.

Comment (IshaanDesai45) [reply public/stylesheets/main.css]: I am so sorry,I completely forgot about that

**Pull request merged**